### DISCUSSION TITLE
Low-level Error: Forced merge caused intersection

### DISCUSSION THREAD

**Discussion by lab313ru** [NationalSecurityAgency/ghidra#5342] [Q&A]
Subj. I've created my own proc module, and this error just happened in one function. I'm trying to understand what causes it and how to fix that. I don't do any manual merging.

**Comment by shamefulCake1** [comment]
I have the same error decompiling an aarch64 Linux .so file.

Does anybody know what it means?

**Comment by ABratovic** [comment]
Looking at the code and comments in cpp\merge.cc a more descriptive error message would be "Forced merge caused Cover intersection" when merging Pcode HighVariables.
If you have a repeatable example of the error it may be worth opening a new Issue to investigate if there's an issue in the aaarch64 processor files.

**Reply by shamefulCake1** [reply]
https://github.com/NationalSecurityAgency/ghidra/issues/8651
